tensorflow/compiler/xla/service/gpu/gpu_conv_runner.h

```diff
@@ -90,28 +95,40 @@ struct GpuConvParams {
 // naturally play well with the typed APIs provided by StreamExecutor; rather
 // than rewriting everything here, just propagate the dynamic typing to one more
 // place by having either a FusedConvRunner or a ConvRunner.
-class MaybeFusedConvRunner {
+class GenericConvRunner {
```

Comment:
I think `MaybeFusedConvRunner` is fine, since the Graph conv runners are also fused, right?

Also the comment above says "...by having either a FusedConvRunner or a ConvRunner". Also add "GraphConvRunner" to the comment.

Reply:
I changed the comment. I think the class name makes sense since FusedConvRunner and GraphConvRunner are distinct.